jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc

```diff
@@ -96,7 +96,8 @@ image::task-lifecycle.png[Image showing the task lifecycle with no content messa
 
 - **Pod restart:** Check if there were any container agent pod restarts around the time the workflow ran. If the pod was restarted around that time it would have resulted in the job not being processed. In such a case, we recommend rerunning the job once again.
 
-You can check the logs for any of the previous runs using the command `kubectl logs -n circleci <full pod name> --previous`.
+You can check the logs for any of the previous runs using the command `kubectl logs $(kubectl get pods -l app=container-agent -o jsonpath='{.items[0].metadata.name}' -n <namespace>) --previous -n <namespace>  >> container-runner.logs
+`.
```

Comment:
Per internal discussion, we should probably just have this as the same command as before, but without the hardcoded `circleci` namespace.

Reply:
@christian-stephen so just `kubectl logs -n <namespace> <full pod name> --previous` ?

Reply:
Sorry it took so long to respond, but yes just as you put.